tests/test_actions_modernizer_django.py

```diff
@@ -0,0 +1,59 @@
+"""
+Tests for Github Actions Django Modernizer Script
+"""
+import os
+import shutil
+import uuid
+from unittest import TestCase
+
+from edx_repo_tools.codemods.django3 import GithubCIDjangoModernizer
+from edx_repo_tools.utils import YamlLoader
+
+
+class TestGithubActionsDjangoModernizer(TestCase):
```

Comment:
we should remove Django's TestCase based tests and instead write test functions since we mostly run `pytest` across our repos